data/webbrowser.yaml

```diff
@@ -142,6 +142,207 @@ supported_os: [Windows,Darwin,Linux]
+    - '%%users.homedir%%/.config/chrome-remote-desktop/chrome-profile/*/GPUCache/*'
+    - '%%users.homedir%%/.config/chrome-remote-desktop/chrome-profile/*/Media Cache/*'
+    - '%%users.homedir%%/.config/chrome-remote-desktop/chrome-profile/Cache/*'
+    - '%%users.homedir%%/.config/chrome-remote-desktop/chrome-profile/PnaclTranslationCache/*'
+    - '%%users.homedir%%/.config/chromium/*/Application Cache/*'
+    - '%%users.homedir%%/.config/chromium/*/Cache/*'
+    - '%%users.homedir%%/.config/chromium/*/GPUCache/*'
+    - '%%users.homedir%%/.config/chromium/*/Media Cache/*'
+    - '%%users.homedir%%/.config/chromium/Cache/*'
+    - '%%users.homedir%%/.config/chromium/PnaclTranslationCache/*'
+    - '%%users.homedir%%/.config/google-chrome-beta/*/Application Cache/*'
+    - '%%users.homedir%%/.config/google-chrome-beta/*/Cache/*'
+    - '%%users.homedir%%/.config/google-chrome-beta/*/GPUCache/*'
+    - '%%users.homedir%%/.config/google-chrome-beta/*/Media Cache/*'
+    - '%%users.homedir%%/.config/google-chrome-beta/Cache/*'
+    - '%%users.homedir%%/.config/google-chrome-beta/PnaclTranslationCache/*'
+    - '%%users.homedir%%/.config/google-chrome/*/Application Cache/*'
+    - '%%users.homedir%%/.config/google-chrome/*/Cache/*'
+    - '%%users.homedir%%/.config/google-chrome/*/GPUCache/*'
+    - '%%users.homedir%%/.config/google-chrome/*/Media Cache/*'
+    - '%%users.homedir%%/.config/google-chrome/Cache/*'
+    - '%%users.homedir%%/.config/google-chrome/PnaclTranslationCache/*'
+    - '%%users.homedir%%/.config/opera/*/Application Cache/*'
+    - '%%users.homedir%%/.config/opera/*/Cache/*'
+    - '%%users.homedir%%/.config/opera/*/GPUCache/*'
+    - '%%users.homedir%%/.config/opera/*/Media Cache/*'
+    - '%%users.homedir%%/.config/opera/Cache/*'
+    - '%%users.homedir%%/.config/opera/PnaclTranslationCache/*'
+    - '%%users.homedir%%/.config/yandex-browser-beta/*/Application Cache/*'
+    - '%%users.homedir%%/.config/yandex-browser-beta/*/Cache/*'
+    - '%%users.homedir%%/.config/yandex-browser-beta/*/GPUCache/*'
+    - '%%users.homedir%%/.config/yandex-browser-beta/*/Media Cache/*'
+    - '%%users.homedir%%/.config/yandex-browser-beta/Cache/*'
+    - '%%users.homedir%%/.config/yandex-browser-beta/PnaclTranslationCache/*'
+    - '%%users.homedir%%/snap/chromium/common/chromium/*/Application Cache/*'
+    - '%%users.homedir%%/snap/chromium/common/chromium/*/Cache/*'
+    - '%%users.homedir%%/snap/chromium/common/chromium/*/GPUCache/*'
+    - '%%users.homedir%%/snap/chromium/common/chromium/*/Media Cache/*'
+    - '%%users.homedir%%/snap/chromium/common/chromium/Cache/*'
+    - '%%users.homedir%%/snap/chromium/common/chromium/PnaclTranslationCache/*'
+  supported_os: [Linux]
+supported_os: [Windows,Darwin,Linux]
+labels: [Browser]
+urls:
+- 'https://artifacts-kb.readthedocs.io/en/latest/sources/webbrowser/ChromeCache.html'
+- 'https://forensicswiki.xyz/wiki/index.php?title=Google_Chrome#Chrome-derived_Browsers'
```

Comment:
this is already linked from the previous article, I opt to remove it

Reply:
Sure, done.